drawBot/drawBotDrawingTools.py

```diff
@@ -58,7 +62,7 @@ def _deprecatedWarningWrapInTuple(txt):
     '10x14'       : (720, 1008),
 }
 
-for key, (w, h) in _paperSizes.items():
+for key, (w, h) in list(_paperSizes.items()):
```

Comment:
list can be removed safely here

Reply:
Well I put it there because I got an exception. I'll double check.

Reply:
Yeah, because the loop adds items to the dict, so without the `list()` there will be an `RuntimeError: dictionary changed size during iteration` exception.

Reply:
oh, my bad you're right!